### DISCUSSION TITLE
Lighthouse Integration with selenium with login session

### DISCUSSION THREAD

**Discussion by ganeshjoe** [GoogleChrome/lighthouse#13492] [Q&A]
Hi,

Anyone who had integrated selenium automation for an e2e user flow with login sessions

**Comment by adamraine** [comment]
We only officially support puppeteer right now. There was a [previous discussion](https://github.com/GoogleChrome/lighthouse/discussions/13326) about using Playwright that might be useful here.

**Reply by ganeshjoe** [reply]
Selenium there are no threads related to the same
